Neos.Neos/Classes/Service/HtmlAugmenter.php

```diff
@@ -79,7 +79,7 @@ protected function getHtmlRootElement($html)
             return null;
         }
         // detect whether loadHTML has wrapped plaintext in a p-tag without asking for permission
-        if ($rootElement instanceof \DOMNodeList && $rootElement->item(0)->tagName === 'p' && substr($html, 0, 2) !== '<p') {
+        if ($rootElement instanceof \DOMNodeList && $rootElement->item(0)->tagName === 'p' && !preg_match('/^<p/ui', $html)) {
```

Comment:
Since
> preg_match() returns 1 if the pattern matches given subject, 0 if it does not, or FALSE if an error occurred
please use a strict comparison here.